app/javascript/groceries/store.ts

```diff
@@ -26,12 +27,6 @@ interface Nameable {
   name: string;
 }
 
-interface StoreAttributes {
-  name?: string;
-  notes?: string;
-  private?: boolean;
-}
```

Comment:
(Nonessential, unrelated change. This is only used in one place; it seemed more appropriate to inline.)